app.ytmdesktop.ytmdesktop.yml

```diff
@@ -0,0 +1,61 @@
+app-id: app.ytmdesktop.ytmdesktop
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
```

Comment:
21.08 is just out, can you try with that instead as you'll have newer software and a longer security fix period.

Reply:
I sent the pr for 21.08. I haven't had time to rebase my apps yet.